src/controllers/authController.js

```diff
@@ -0,0 +1,176 @@
+const User = require('../models/user.js');
+const userService = require('../services/userService.js');
+const tokenService = require('../services/tokenService.js');
+const jwtService = require('../services/jwtService.js');
+const ApiError = require('../exeptions/apiError.js');
+const bcrypt = require('bcrypt');
+
+const validateEmail = (value) => {
+  if (!value) {
+    return 'Email is required';
+  }
+
+  const emailPattern = /^[\w.+-]+@([\w-]+\.){1,3}[\w-]{2,}$/;
+
+  if (!emailPattern.test(value)) {
+    return 'Email is not valid';
+  }
+};
+
+const validaPassword = (value) => {
```

Comment:
There is a typo in the function name `validaPassword`. It should be `validatePassword` to maintain consistency and clarity.